ls1intum/Artemis
# Show student name within instructor assessment dashboards

### Checklist

### Motivation and Context


Currently within the exercise specific assessment dashboards for file upload exercises and text exercises and programming exercises no student names are shown. In this PR we make the handling uniform across all exercise types.

### Description

Student names are now shown for file upload exercises and text exercises and programming exercises within the specific assessment dashboards. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> exercises -> submissions
3. you should see students names 

### Test Coverage




### Screenshots


![image](https://user-images.githubusercontent.com/33342534/110252038-cb1c3980-7f83-11eb-88c4-a6f5cc9b9aa0.png)
![image](https://user-images.githubusercontent.com/33342534/110252053-d53e3800-7f83-11eb-8238-75e47d01dd06.png)
![image](https://user-images.githubusercontent.com/33342534/110252061-dcfddc80-7f83-11eb-89db-62e8e665e92c.png)
![image](https://user-images.githubusercontent.com/33342534/110252068-e5eeae00-7f83-11eb-9ab0-b61ddced84e3.png)
